concourse/pipeline.yml

```diff
@@ -1878,16 +1878,125 @@ jobs:
+    params:
+      <<: *ci_security_user
+      <<: *ci_service_endpoints_for_python
+      MAKE_TARGET: setup
+  - task: rasrm-acceptance-tests
+    on_failure:
+      put: notify
+      params:
+        text: "RASRM acceptance tests failed in CI space"
+    privileged: true
+    file: ras-deploy/tasks/rasrm-acceptance-tests/run_acceptance_tests.yml
+    params:
+      <<: *ci_security_user
+      <<: *ci_service_endpoints_for_python
+      MAKE_TARGET: rasrm_acceptance_tests
+
+- name: ci-securemessage-acceptance-tests
+  serial_groups:
+    *ci_all_services
+  plan:
+  - get: ras-deploy
+  - get: ci-teardown-timer
+    trigger: true
+    passed: *ci_all_services
+  - get: rasrm-acceptance-tests-source
+    trigger: true
+  - get: ras-party-source
+    passed: [ras-party-ci-deploy]
+    trigger: true
+  - get: ras-backstage-source
+    passed: [ras-backstage-ci-deploy]
+    trigger: true
+  - get: ras-frontstage-source
+    passed: [ras-frontstage-ci-deploy]
+    trigger: true
+  - get: ras-frontstage-api-source
+    passed: [ras-frontstage-api-ci-deploy]
+    trigger: true
+  - get: django-oauth2-test-source
+    passed: [django-oauth2-test-ci-deploy]
+    trigger: true
+  - get: response-operations-ui-source
+    passed: [response-operations-ui-ci-deploy]
+    trigger: true
+  - get: ras-secure-message-source
+    passed: [ras-secure-message-ci-deploy]
+    trigger: true
+  - get: ras-collection-instrument-source
+    passed: [ras-collection-instrument-ci-deploy]
+    trigger: true
+  - get: rm-action-service-source
+    passed: [rm-action-service-ci-deploy]
+    trigger: true
+  - get: rm-actionexporter-service-source
+    passed: [rm-actionexporter-service-ci-deploy]
+    trigger: true
+  - get: rm-case-service-source
+    passed: [rm-case-service-ci-deploy]
+    trigger: true
+  - get: rm-collection-exercise-service-source
+    passed: [rm-collection-exercise-ci-deploy]
+    trigger: true
+  - get: iac-service-source
+    passed: [iac-service-ci-deploy]
+    trigger: true
+  - get: rm-notify-gateway-source
+    passed: [rm-notify-gateway-ci-deploy]
+    trigger: true
+  - get: rm-sample-service-source
+    passed: [rm-sample-service-ci-deploy]
+    trigger: true
+  - get: rm-sdx-gateway-source
+    passed: [rm-sdx-gateway-ci-deploy]
+    trigger: true
+  - get: rm-survey-service-source
+    passed: [rm-survey-service-ci-deploy]
+    trigger: true
+  - get: sdc-uaa-source
+    passed: [sdc-uaa-ci-deploy]
+    trigger: true
+  - task: get-cf-database-uris
+    file: ras-deploy/tasks/get_database_uris.yml
+    params:
+      CLOUDFOUNDRY_API: ((cloudfoundry_api))
+      CLOUDFOUNDRY_EMAIL: ((cloudfoundry_email))
+      CLOUDFOUNDRY_PASSWORD: ((cloudfoundry_password))
+      CLOUDFOUNDRY_ORG: rmras
+      CLOUDFOUNDRY_SPACE: ci
+      MAKE_TARGET: rasrm_acceptance_tests
+  - task: acceptance-test-setup
+    on_failure:
+      put: notify
+      params:
+        text: "Acceptance tests setup failed in CI space"
```

Comment:
Might be worth changing this to indicate it's the secure messaging tests that failed?